packages/core/src/util/getPreloadFlags.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+exports.getPreloadFlags = function getPreloadFlags() {
```

Comment:
issue: I think it would have been better to add a quick unit test. The function now prints:

> $ INSTANA_DEBUG=true node --require '@instana/collector/src/immediate.js' ./dummy-app/src/index.js
> The App is using the following preload flags: --require

I expected to see:

> The App is using the following preload flags: --require @instana/collector/src/immediate.js

We need the type + the value.

And as discussed:

> const instanaKeyword = '@instana';

We don't really need that? We can just print all --require and all --import flags etc. That will help debugging conflicts with other preload flags. Or why did you re-add this again?

Reply:
> We don't really need that? We can just print all --require and all --import flags etc. 
> 
I thought of filtering our packages alone. No other reasons. But as you mentioned we can print all the preloaded packages because that can help debug the conflicts. I will update the  code as per the suggestion